model/patch_lifecycle.go

```diff
@@ -211,11 +209,7 @@ func MakePatchedConfig(ctx context.Context, p *patch.Patch, remoteConfigPath, pr
 		if err != nil {
 			return nil, errors.Wrap(err, "could not read patched config file")
 		}
-		project := &Project{}
-		if err = LoadProjectInto(data, p.Project, project); err != nil {
-			return nil, errors.WithStack(err)
-		}
-		return project, nil
+		return data, nil
```

Comment:
now we we unmarshall and validate the patch in `validator.ValidateProjectPatch(...)`